apps/rod/sections/summary-data-sections.js

```diff
@@ -20,46 +20,59 @@ module.exports = {
       step: '/sponsor-type',
       field: 'sponsor-type'
     },
+    {
+      step: '/application',
+      field: 'application-type'
+    },
+    {
+      step: '/visa-type',
+      field: 'visa-type'
+    },
     {
       step: '/about-application',
       field: 'date-of-application',
       parse: d => d && dateFormater.format(new Date(d))
     },
+
     {
       step: '/about-application',
       field: 'cancel-application'
     },
     {
-      step: '/dependant-or-guardian',
-      field: 'dependant-or-guardian'
-    },
-    {
-      step: '/legal-representation',
-      field: 'legal-rep-name'
+      step: '/main-applicant-passport',
+      field: 'is-requesting-passport-to-travel'
     },
     {
-      step: '/application',
-      field: 'application-type'
+      step: '/further-leave',
+      field: 'further-leave-to-remain'
```

Comment:
I'm not sure if this order is correct. I reckon further leave to remain section would show after the visa type